src/status_im2/contexts/profile/login/events.cljs

```diff
@@ -1,3 +1,75 @@
-(ns status-im2.contexts.profile.login.events)
+(ns status-im2.contexts.profile.login.events
+  (:require [utils.re-frame :as rf]
+            [status-im.ethereum.core :as ethereum]
+            [utils.security.core :as security]
+            [re-frame.core :as re-frame]
+            [native-module.core :as native-module]
+            [status-im2.navigation.events :as navigation]
+            [status-im2.common.keychain.events :as keychain]
+            [status-im2.common.biometric.events :as biometric]
+            [status-im2.contexts.profile.config :as profile.config]
+            [utils.transforms :as transforms]))
 
+(re-frame/reg-fx
+ ::login
+ (fn [[key-uid hashed-password]]
+   ;;"node.login" signal will be triggered as a callback
+   (native-module/login-account
+    (assoc (profile.config/login) :keyUid key-uid :password hashed-password))))
 
+(rf/defn login
+  {:events [:profile.login/login]}
+  [{:keys [db]}]
+  (let [{:keys [key-uid password]} (:profile/login db)]
+    {:db     (assoc-in db [:profile/login :processing] true)
+     ::login [key-uid (ethereum/sha3 (security/safe-unmask-data password))]}))
+
+(rf/defn login-local-paired-user
+  {:events [:profile.login/local-paired-user]}
+  [{:keys [db]}]
+  (let [{:keys [key-uid name password]} (get-in db [:syncing :profile])]
+    {::login [key-uid
+              (transforms/clj->json {:name    name
```

Comment:
@flexsurfer - You should remove the `(transforms/clj->json {:name name :key-uid key-uid})` as the `::login` accepts only `key-uid` and `hashed-password`.

This is probably why the pairing is struck on the `Syncing devices...` screen as we are not passing the password to the login method correctly.